### PR TITLE
magento/devdocs#4649: Topic clarification/correction request

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) provides a fix for https://github.com/magento/devdocs/issues/4649.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/index.html
- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/data-patches.html

## Links to Magento source code

- ...

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/extension-dev-guide/declarative-schema/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Once a module is converted to the declarative schema approach, it cannot be reve
 
 * *Data patch* - A class that contains data modification instructions. It can have dependencies on other data or schema patches.
 
-* *Revertable data patch* - A patch that can be reverted as a module or patch is uninstalled or deleted. Revertable operations are Data Query Language (DQL) and Data Manipulation Language (DML) operations: INSERT, UPDATE.
+* *Revertable data patch* - A `data patch` that contains a `revert()` method (see [Reverting data patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html/#revertingDataPatches)) with provided revertable operations.
```
Comment (rogyar) [inline guides/v2.3/extension-dev-guide/declarative-schema/index.md]: We need to reconsider the meaning of "revertable operations" here. The current change says that the `revert()` method contains **revertable** operations. But in fact the `revert()` method contains **revert** operations. 
**Revertable** operations - operations that can be reverted. So it's not 100% correct to say that `revert()` method contains operations that can be reverted. It contains operations that revert revertable changes caused by the module. 
Also, I would provide some infomation about what changes are revertable (changes in the database) and what are not (changes in the core code).

File: guides/v2.3/extension-dev-guide/declarative-schema/index.md [hunk 2]
```diff
@@ -40,3 +40,5 @@ Once a module is converted to the declarative schema approach, it cannot be reve
   * Performing data migration with inside DDL operations
   * Renaming tables, columns, and other entities
   * Adding partitions and options to a table
+
+* *Revertable schema patch* - A `schema patch` that contains a `revert()` method with provided revertable functionality.
```
Comment (rogyar) [inline guides/v2.3/extension-dev-guide/declarative-schema/index.md]: My previous comment about revertable changes is fair for this conclusion as well.

File: guides/v2.3/extension-dev-guide/declarative-schema/index.md (hunk 1 above)
Comment (atwixfirster) [reply guides/v2.3/extension-dev-guide/declarative-schema/index.md]: Thank you for your notice, @rogyar !

I've implemented your suggestions. Could you please verify a new solution?

Thank you!

File: guides/v2.3/extension-dev-guide/declarative-schema/index.md (hunk 2 above)
Comment (atwixfirster) [reply guides/v2.3/extension-dev-guide/declarative-schema/index.md]: **fixed**

Could you please verify a new fix?

Thank you!

File: guides/v2.3/extension-dev-guide/declarative-schema/index.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ Once a module is converted to the declarative schema approach, it cannot be reve
 
 * *Data patch* - A class that contains data modification instructions. It can have dependencies on other data or schema patches.
 
-* *Revertable data patch* - A patch that can be reverted as a module or patch is uninstalled or deleted. Revertable operations are Data Query Language (DQL) and Data Manipulation Language (DML) operations: INSERT, UPDATE.
+* *Revertable data patch* - A `data patch` that contains a `revert()` method (see [Reverting data patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html/#revertingDataPatches)) with provided operations that revert revertable changes caused by the module. Revertable operations usually related to the changes in the database.
```
Comment (jeff-matthews) [inline guides/v2.3/extension-dev-guide/declarative-schema/index.md]:
```suggestion
* *Revertable data patch* - A `data patch` that contains a `revert()` method (see [Reverting data patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html/#revertingDataPatches)) with provided operations that revert revertable changes caused by the module. Revertable operations are usually related to changes in the database.
```

Comment (atwixfirster) [reply guides/v2.3/extension-dev-guide/declarative-schema/index.md]: **fixed**

Thank you, @jeff-metthews!

**Pull request merged**